cmd/fleetctl/package_test.go

```diff
@@ -43,10 +45,45 @@ func TestPackage(t *testing.T) {
 	}
 
 	t.Run("deb", func(t *testing.T) {
-		runAppForTest(t, []string{"package", "--type=deb", "--insecure", "--disable-open-folder"})
-		info, err := os.Stat(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
-		require.NoError(t, err)
-		require.Greater(t, info.Size(), int64(0)) // TODO verify contents
+		shorterEnrollSecret := "aa"
+		longerEnrollSecret := "aaaaaa"
+
+		checkEnrollSecret := func(t *testing.T, expectedEnrollSecret string, updatesData *packaging.UpdatesData) {
+			runAppForTest(t, []string{"package", "--type=deb", "--insecure", "--disable-open-folder", "--enroll-secret=" + expectedEnrollSecret, "--fleet-url=https://localhost:8080"})
+			info, err := os.Stat(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
+			require.NoError(t, err)
+			require.Greater(t, info.Size(), int64(0))
+
+			fd, err := os.Open(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
+			require.NoError(t, err)
+
+			defer fd.Close()
+
+			debFile, err := deb.Load(fd, fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
+			require.NoError(t, err)
+
+			for {
+				hdr, err := debFile.Data.Next()
+				if err == io.EOF {
+					break
+				}
+				require.NoError(t, err)
+
+				if hdr.Name != "./etc/default/orbit" {
+					continue
+				}
+
+				data, err := io.ReadAll(debFile.Data)
+				require.NoError(t, err)
+
+				s := string(data)
+				require.Contains(t, s, fmt.Sprintf("ORBIT_ENROLL_SECRET=%s\n", expectedEnrollSecret))
+				t.Log("we did it")
+			}
+		}
+
+		checkEnrollSecret(t, shorterEnrollSecret, updatesData)
+		checkEnrollSecret(t, longerEnrollSecret, updatesData)
 	})
```

Comment:
Shouldn't it be longer first?

Reply:
that's what I get for multi tasking... 🤦